src/components/AppProvider/README.md

```diff
@@ -350,11 +350,15 @@ class ProviderThemeExample extends React.Component {
 
 ## Initializing the Shopify App Bridge
 
-When using Polaris, you don’t need to go through the initialization of the Shopify App Bridge as described [in the docs](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app). Instead, configure the connection to the Shopify admin through the app provider component, which must wrap all components in an embedded app. This component initializes the Shopify App Bridge using the `apiKey` you provide. **The `apiKey` attribute is required** and can be found in the [Shopify Partner Dashboard](https://partners.shopify.com).
+When using Polaris, you don’t need to go through the initialization of the Shopify App Bridge as described in the [Shopify Help Center](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app). Instead, configure the connection to the Shopify admin through the [app provider component](https://polaris.shopify.com/components/structure/app-provider), which wraps all components in an embedded app. The app provider component initializes the Shopify App Bridge using the `apiKey` and `shopOrigin` that you provide.
+
+**The `apiKey` attribute is required**. You can find the API key for your app in the Apps section of your [Shopify Partner Dashboard](https://partners.shopify.com).
+
+**The `shopOrigin` attribute is required**. Learn how to get and store the shop origin in the [Shopify Help Center](https://help.shopify.com/en/api/embedded-apps/shop-origin).
```

Comment:
Based on @gf3's comment, we should state `shopOrigin` as required field despite the props contract.

Reply:
Do you have any thoughts before shipping this PR? @sadiesaurus @selenehinkley

Reply:
Hey @henrytao-me! Sadie and I edited that last bit to condense it, while still getting the message accross that the `apiKey` and `shopOrigin` are required. If you like what we updated it to, it's all good to 🚢